### PR TITLE
Fixed createdAt updateAt in created location

### PR DESCRIPTION
**Pull request by nikitius20**

### CODE REVIEW

File: backend/models/Locations.ts [hunk 1]
```diff
@@ -40,6 +40,9 @@ const schema = new mongoose.Schema({
   author: { type: String, required: true },
   createdAt: { type: Date },
   updatedAt: { type: Date }
+},
```
Comment (6mahdapihka9) [inline backend/models/Locations.ts]: Is it necessary to keep these two fields in the scheme declaration, in case you've already declared the timestamp?

**Review by Yojuks — APPROVED** [review]
There was before removing those code)

**Review by 6mahdapihka9 — APPROVED** [review]
have you tried to post new location?

File: backend/models/Locations.ts (hunk 1 above)
Comment (nikitius20) [reply backend/models/Locations.ts]: no it is not necessary, i removed it. i also removed it from the interface, and  it works

**Comment by nikitius20** [comment]
> have you tried to post new location?

yes it works

**Pull request merged**